contracts/NewfiAdvisor.sol

```diff
@@ -71,6 +72,8 @@ contract NewfiAdvisor is
 
     address[] public advisors;
 
+    address[] public advisorTokens;
```

Comment:
do we need this ? we have the struct where we track the advisors both pool tokens

Reply:
It's a list of all the tokens in existence, might not be super helpful though. Dunno